cli/azd/cmd/util.go

```diff
@@ -236,13 +236,21 @@ func getResourceGroupFollowUp(
 	projectConfig *project.ProjectConfig,
 	resourceManager project.ResourceManager,
 	env *environment.Environment,
+	whatIf bool,
 ) (followUp string) {
 	if formatter.Kind() != output.JsonFormat {
 		subscriptionId := env.GetSubscriptionId()
 
 		if resourceGroupName, err := resourceManager.GetResourceGroupName(ctx, subscriptionId, projectConfig); err == nil {
-			followUp = fmt.Sprintf("You can view the resources created under the resource group %s in Azure Portal:\n%s",
-				resourceGroupName, output.WithLinkFormat(fmt.Sprintf(
+			defaultFollowUpText := fmt.Sprintf(
+				"You can view the resources created under the resource group %s in Azure Portal:", resourceGroupName)
+			if whatIf {
+				defaultFollowUpText = fmt.Sprintf(
+					"You can view the current resources under the resource group %s in Azure Portal:", resourceGroupName)
```

Comment:
I wonder if long term we would want to remove this message from `--preview`.  I'm not sure pointing folks at the RG in this case every time provides a lot of value.  Perhaps we have have an `azd show` targeted at humans and not computers (e.g. the `--output=json` mode that we support today for editors) we can remove this.

Just a note - non blocking feedback.